pages/rideDetailsStory.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+import {withKnobs, text, boolean, select} from '@storybook/addon-knobs'
+
+import { Column } from '_utils/layout'
+import ItemInfo from 'itemInfo'
+import ItemData from 'itemData'
+import Divider from 'divider'
+import TheVoice from 'theVoice'
+import BubbleIcon from 'icon/bubbleIcon'
+import SmokeIcon from 'icon/smokeIcon'
+import PetIcon from 'icon/petIcon'
+import { status } from '_utils/icon/status'
+import ItemAction from 'itemAction'
+import SubHeader from 'subHeader'
+import Itinerary from 'itinerary'
+import ItemChoice from 'itemChoice'
+import Avatar from 'avatar'
+
+const stories = storiesOf('Pages|Ride details', module)
+stories.addDecorator(withKnobs)
+
+stories.add('default', () => (
+    <div style={{width: '600px', margin: 'auto', border: 'lightgray 1px solid'}}>
+        <Column>
+            <TheVoice>Ven. 11 octobre</TheVoice>
+            <Itinerary
+                fromAddon='Paris'
+                toAddon='Marseille'
+                places={[
+                    {
+                        mainLabel: 'Paris',
+                        isoDate: '2017-12-11T09:00',
+                        time: '09:00',
+                        href: '#',
+                    },
+                    {
+                        mainLabel: 'Marseille',
+                        isoDate: '2017-12-11T12:00',
+                        time: '17:00',
+                    },
+                ]}
+                small={boolean('small', false)}
```

Comment:
Not sure we want to keep this knob. Or maybe name it like "[Itinerary] small" so we know which component is affected

Reply:
Can we group knobs? (I don't think so, but since we now have pages in ui-library, it would be cool if we could)